include/tesseract/baseapi.h

```diff
@@ -531,6 +532,28 @@ class TESS_API TessBaseAPI {
    * as UTF8 and must be freed with the delete [] operator.
    */
   char *GetUTF8Text();
+  
+  size_t GetNumberOfTables();
+  
+  /// Return the i-th table bounding box coordinates
+  ///
+  ///Gives the (top_left.x, top_left.y, bottom_right.x, bottom_right.y)
+  /// coordinates of the i-th table.
+  std::tuple<int,int,int,int> GetTableBoundingBox(
+    unsigned i///< Index of the table, for upper limit \see GetNumberOfTables()
```

Comment:
but here we have unsigned, not size_t (different sizes on 32 and 64 bit platforms)